src/main/asciidoc/docinfo-footer.html

```diff
@@ -0,0 +1,51 @@
+<script type="text/javascript" src="tocbot-3.0.2/tocbot.js"></script>
```

Comment:
Should these additions (also `docinfo.html` below) rather be done in the parent project and be consumed by the build? I'd want to avoid having to add these files to all Spring Data projects as that makes changes to them cumbersome.